### PR TITLE
DM-27086: Numpydoc conversion of meas_algorithms through gaussianPsfFactory.py

### PR DESCRIPTION
**Pull request by squisty**

### CODE REVIEW

File: python/lsst/meas/algorithms/gaussianPsfFactory.py [hunk 1]
```diff
@@ -154,7 +167,10 @@ def makeField(cls, doc):
         def applyWrapper(config, **kwargs):
             """Construct a Gaussian PSF
 
-            @param[in] config: an instance of GaussianPsfFactory
+            Parameters
+            ----------
+            config : ``lsst.pex.config.ConfigurableField``
```
Comment (parejkoj) [inline python/lsst/meas/algorithms/gaussianPsfFactory.py:172]: The type here is `GaussianPsfFactory` (this class). I can see the confusion: this is a closure that is used by `ConfigurableField` to construct the correct kind of PSF, and so we're not documenting the return of `makeField` here. Keep the docstring as-written (even though it's sort of redundant).

File: doc/lsst.meas.algorithms/tasks/lsst.meas.algorithms.FlaggedSourceSelectorTask.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+``FlaggedSourceSelectorTask`` uses an existing flag field to filter
+a SourceCatalog.
+
+.. _lsst.meas.algorithms.FlaggedSourceSelectorTask-summary:
+
+Processing summary
+==================
+
+``FlaggedSourceSelectorTask`` allows procedures that need Sources
+to use the Sources to determine the PSF.
```
Comment (parejkoj) [inline doc/lsst.meas.algorithms/tasks/lsst.meas.algorithms.FlaggedSourceSelectorTask.rst:16]: Lets change that to "to use those Sources", which I think clarifies the intent.

**Review by parejkoj — CHANGES_REQUESTED** [review]
Thanks: a few comments, including correcting a non-building numbered list.

File: doc/lsst.meas.algorithms/tasks/lsst.meas.algorithms.DynamicDetectionTask.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+.. _lsst.meas.algorithms.DynamicDetectionTask-summary:
+
+Processing summary
+==================
+
+``DynamicDetectionTask`` runs this sequence of operations:
+
+#. Detects sources using a lower threshold than normal in order
+to identify good sky regions.
```
[the remark this answers is not in the document]
Comment (squisty) [reply doc/lsst.meas.algorithms/tasks/lsst.meas.algorithms.DynamicDetectionTask.rst:17]: Fixed the sentence issue, the template for the task topic mentioned on https://developer.lsst.io/stack/task-topic-type.html#starter-template has blank lines in between so I will go off of that for now?

[the remark this answers is not in the document]
Comment (parejkoj) [reply doc/lsst.meas.algorithms/tasks/lsst.meas.algorithms.DynamicDetectionTask.rst]: The numbered list generated correctly now.

**Review by parejkoj — APPROVED** [review]

**Pull request merged**